content/_data/authors/abhishek_gautam.adoc

```diff
@@ -0,0 +1,12 @@
+---
+name: Abhishek Gautam
+github: "gautamabhishek46"
+IRC nick: "abhishekg"
+---
+
+Abhishek is a 3rd year Computer Science student from Visvesvaraya National
+Institute of Technology, Nagpur, India. He has done some website projects for
+his college technical festival. He is also a regular competitive programmer
+(abhishekg1128 at codechef). He has done two internships as a Game Programmer
+as well. He was a member of ACM Chapter and Google student developer club of his
+college. He joined Jenkins through GSOC 2018, as is intrested in automation.
```

Comment:
I would rephrase the last sentence to something like: His interest in automation motivated his participation in the Jenkins GSOC 2018 program.